Gemfile

```diff
@@ -9,22 +9,24 @@ gem 'bootstrap-will_paginate', '0.0.10'
 gem 'bootstrap_form', '2.5.2'
 gem 'chartkick', '2.2.1' # Chart project_stats
 gem 'faker', '1.6.6'
-gem 'fastly-rails', '0.7.1' # Use Fastly CDN
+gem 'fastly-rails', git: 'https://github.com/dankohn/fastly-rails',
+                    ref: 'a909d09'
```

Comment:
I *really* appreciate that you're fixing problems in gems... but I hate that you need to do it.  If they fix other stuff, this may be hard to deal with.

I see that this version:
https://github.com/dankohn/fastly-rails/commit/a909d09d23a3ea5af0a89af175ea47bf490566fd

Removes the railties dependency spec: 
~~~~
s.add_dependency "railties", '< 5'
~~~~

Have you tried to get this upstream?